code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer_reactions.dm

```diff
@@ -0,0 +1,52 @@
+GLOBAL_LIST_INIT(electrolyzer_reactions, electrolyzer_reactions_list())
+
+/*
+ * Global proc to build the electrolyzer reactions list
+ */
+/proc/electrolyzer_reactions_list()
+	. = list()
+	for(var/reaction_path in subtypesof(/datum/electrolyzer_reaction))
+		var/datum/electrolyzer_reaction/reaction = new reaction_path()
+
+		.[reaction.id] = reaction
```

Comment:
Should just use a list var here rather then .
. is appropriate for scaring newfriends and making very hot code fast, but it's not needed here